website/docs/r/nrql_alert_condition.html.markdown

```diff
@@ -48,81 +60,166 @@ See additional [examples](#additional-examples).
 
 The following arguments are supported:
 
+- `account_id` - (Optional) The New Relic account ID of the account you wish to create the condition. Defaults to the account ID set in your environment variable `NEWRELIC_ACCOUNT_ID`.
+- `baseline_direction` - (Optional) The baseline direction of a _baseline_ NRQL alert condition. Valid values are: `lower_only`, `upper_and_lower`, `upper_only` (case insensitive).
```

Comment:
🤔💭 This is a case-insensitive, however, should we provide examples with the casing that matches NerdGraph (uppercase)? Or should we show examples with everything as lowercase. 

Note: We can only make this decision for new attributes. The old attributes must stay the same as before due to the `outlier` condition type.

Reply:
I think this is fine.